src/shipit_static_analysis/shipit_static_analysis/report/mail.py

```diff
@@ -11,10 +11,21 @@
 logger = log.get_logger(__name__)
 
 
-EMAIL_HEADER = '''{nb_publishable} Publishable issues on Mozreview
+EMAIL_STATS_LINE = '|{source}|{total}|{publishable}|'
+
+EMAIL_HEADER = '''
+# Found {total} issues
```

Comment:
Nit: As before, I find the number of `{publishable}` issues more important than `{total}` issues (because we only care about what will get published to the code review service). Total issues are nice to know / be aware of, but not particularly relevant to this code review bot (yet). Please change your title to something like `Found {publishable} publishable issues` or `{publishable} issues to be reported` or similar.